package.json

```diff
@@ -19,13 +19,25 @@
   },
   "scripts": {
     "start": "react-scripts start",
+    "lint:write": "./node_modules/.bin/eslint --debug --fix",
```

Comment:
lint:fix ?

Reply:
I've changed the script name to lint:fix